templates/connect-inject-deployment.yaml

```
@@ -83,6 +83,11 @@ spec:
               CONSUL_FULLNAME="{{template "consul.fullname" . }}"
 
               consul-k8s inject-connect \
+                {{- if .Values.connectInject.logLevel }}
+                -log-level={{ .Values.connectInject.logLevel }} \
+                {{- else }}
+                -log-level={{ .Values.global.logLevel }} \
+                {{- end }}
```

```suggestion
                -log-level={{ default .Values.global.logLevel .Values.connectInject.logLevel }} \
```
I think we can use `default` for anywhere we're doing this style

---

`-log-json` isn't passed